### ISSUE TITLE
cSploit 2 Request

### ISSUE THREAD

**Issue by bomboem** [cSploit/android#880]
Hello.
I know you guys are working hard and have other stuff to do and can't develop around the clock...
I know.

BUT i love csploit (1), and it's getting a little old. Chrome even gives a certificate warning in which csploit is exposed (Cert: cSploit)

We are hungry for cSploit 2.

PLEASE Give us a pre-alpha or even a pre-alpha-alpha, but give us something!

You guys are awesome!

**Comment by Silur** [comment]
Thanks for your input

We realize that the delay of cSploit2 is making an effect on the community, I think the team is mostly waiting for me with the frontend, so 
_I hereby promise a full frontend catch-up to the current (2017-06-14) state of the backend until **Aug 14**_
After that it shouldn't take much time to assemble an alpha :)

**Comment by jboss38** [comment]
@Silur If you need help, ur community is here :)

**Comment by bomboem** [comment]
@Silur Thank you for the quick responce! Really. I've just donated the project.
Maybe one little more thing (and i'll stop nagging :P )
Could you update the "News" page @ csploit.org ? It's a little bit outdated.

Thanks a lot for all the time and power you put in this project.
If you need anything, like anything, your community is here for you!

**Comment by Silur** [comment]
yes I've notified the team on slack about this personal deadline of mine, we'll add it to the site

**Comment by BearChavez** [comment]
@Silur Would you happen to be able to train me a little in developing this?

I'd love to help out, especially with porting to iOS. So many people laugh at pentesting and such on iOS, and my mission is to prove them wrong. 

I wouldn't need one on one training, just links to documents and such would be enough

**Comment by maddog9696** [comment]
While I cant speak for everyone, most of these developers are very busy. They have very little time to even work on csploit2. 

Speaking of which, @Silur you promised August 14th. You surprisingly put it in writing, so now you gotta stick to it! :) Perhaps you've finished and are waiting on the rest of the team now? Can we get an update??

**Comment by Silur** [comment]
@BearChavez sure, join us on slack and we can discuss about the iOs testing
@maddog9696 yes the catch-up is done in functionality (this means a completely functional network-radar and portmap), only some CSS swag is remaining we will produce an alpha preview APK soon.

**Comment by BearChavez** [comment]
@Silur Email me info. 

bearofworlds@hotmail.com

Thanks for the invitation
